schema/application/resolvers/Mutation/saveChecklist.ts

```diff
@@ -1187,6 +1199,8 @@ async function saveChecklistResults(
                             )
                             AND custagtype = ${(() => {
                               switch (true) {
+                                case "boolean" in i.result.widget:
+                                  return "Boolean";
                                 case "checkbox" in i.result.widget:
                                   return "Boolean";
```

Comment:
When you get around to it (creating the new custag), these will need to change to "Checkbox".

Reply:
ditto circa L1163 in this file